docs/requirements.txt

```diff
@@ -1,2 +1,4 @@
+m2r2
```

Comment:
Do you need to include `json` here?

Reply:
`json` is part of the core python package.  Just verified this compiles correctly on RTD: https://pysatmissions.readthedocs.io/en/doc-2021_apr/